addons/mail/static/src/components/notification_list/notification_list.js

```diff
@@ -44,35 +39,9 @@ export class NotificationList extends Component {
      * @returns {Object[]}
      */
     get notifications() {
-        const { notifications } = this.storeProps;
-        return notifications;
-    }
-
-    //--------------------------------------------------------------------------
-    // Private
-    //--------------------------------------------------------------------------
-
-    /**
-     * Load previews of given thread. Basically consists of fetching all missing
-     * last messages of each thread.
-     *
-     * @private
-     */
-    async _loadPreviews() {
-        const threads = this.notifications
-            .filter(notification => notification.thread && notification.thread.exists())
-            .map(notification => notification.thread);
-        this.env.models['mail.thread'].loadPreviews(threads);
-    }
-
-    /**
-     * @private
-     * @param {Object} props
-     */
-    _useStoreSelector(props) {
-        const threads = this._useStoreSelectorThreads(props);
+        const threads = this._useStoreSelectorThreads(this.props);
```

Comment:
Could take the opportunity to rename `_useStoreSelectorThreads` so there's no more reference to `useStore` after this PR.

Reply:
I'll do

Reply:
There were 2 other occurrences of `useStore` in comments too, I also removed them (one I reworded, the other was on a fixme that is now obsolete).